gatling-http/src/main/scala/io/gatling/http/request/builder/Http.scala

```diff
@@ -27,15 +27,15 @@ import io.netty.handler.codec.http.HttpMethod
  */
 final class Http(requestName: Expression[String]) {
   def get(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.GET, url)
-  def get(uri: Uri): HttpRequestBuilder = httpRequest(HttpMethod.GET, Right(uri))
+  def get(uri: Uri): HttpRequestBuilder = httpRequest(Right(HttpMethod.GET), Right(uri))
   def put(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.PUT, url)
   def post(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.POST, url)
   def patch(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.PATCH, url)
   def head(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.HEAD, url)
   def delete(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.DELETE, url)
   def options(url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.OPTIONS, url)
-  def httpRequest(method: String, url: Expression[String]): HttpRequestBuilder = httpRequest(HttpMethod.valueOf(method), Left(url))
-  def httpRequest(method: HttpMethod, url: Expression[String]): HttpRequestBuilder = httpRequest(method, Left(url))
-  def httpRequest(method: HttpMethod, urlOrURI: Either[Expression[String], Uri]): HttpRequestBuilder =
+  def httpRequest(method: Expression[String], url: Expression[String]): HttpRequestBuilder = httpRequest(Left(method), Left(url))
```

Comment:
This is a binary breaking change. You cannot change existing signatures.

Honestly, I've tried. And failed.
If things were that simple, I would have done it myself.
I ended up with the Scala compiler no longer being able to pick my implicit conversions, causing tests to no longer compile.